maplibre_gl_web/lib/src/maplibre_web_gl_platform.dart

```diff
@@ -1,9 +1,5 @@
 part of maplibre_gl_web;
 
-//TODO Url taken from the Maptiler tutorial; use official and stable release once available
-final _maplibreGlCssUrl =
-    'https://cdn.maptiler.com/maplibre-gl-js/v1.13.0-rc.4/mapbox-gl.css';
```

Comment:
So this is now available via the 4.3 dist stylesheet?

Reply:
My guess was that injecting it here was a workaround in the times of early stage dart js interop. I tried to figure out some information about the problem back then but wasn't successful. The example app works the same without it and I couldn't find a reason leaving it in.